src/Presentation/Presenters/DonationConfirmationHtmlPresenter.php

```diff
@@ -29,25 +29,25 @@ public function __construct( TwigTemplate $template, UrlGenerator $urlGenerator,
 		$this->validation = $validation;
 	}
 
-	public function present( Donation $donation, string $updateToken, string $accessToken,
+	public function present( Donation $donation, array $paymentData, string $updateToken, string $accessToken,
 							 array $urlEndpoints ): string {
 		return $this->template->render(
-			$this->getConfirmationPageArguments( $donation, $updateToken, $accessToken, $urlEndpoints )
+			$this->getConfirmationPageArguments( $donation, $paymentData, $updateToken, $accessToken, $urlEndpoints )
 		);
 	}
 
-	private function getConfirmationPageArguments( Donation $donation, string $updateToken, string $accessToken,
+	private function getConfirmationPageArguments( Donation $donation, array $paymentData, string $updateToken, string $accessToken,
 		array $urlEndpoints ): array {
 		$donorDataFormatter = new DonorDataFormatter();
 		return [
 			'donation' => [
 				'id' => $donation->getId(),
-				'amount' => $donation->getAmount()->getEuroFloat(),
-				'interval' => $donation->getPaymentIntervalInMonths(),
-				'paymentType' => $donation->getPaymentMethodId(),
+				'amount' => $paymentData['amount'],
```

Comment:
Danger! The unfortunate frontend expects euro floats, not euro cents! My suggestion for a seamless experience would be to convert the amount to float using the `Euro` class, add a `amountInCents` property and later adapt the frontend to use cents.

Reply:
We'll also need that to do currency localisation